.github/ISSUE_TEMPLATE/new_feature

```diff
@@ -0,0 +1,13 @@
+## Desired Behavior
+
+Describe desired behavior for improvements/enhancements.
+What is the problem we want to solve?
+What is the plan?
+
+## Actual Behavior
```

Comment:
Although there's always a way to fill it somehow, I feel it's not always relevant for new features, so I would also suggest this section to be optional as well, not a blocker tho.

Reply:
Good point, done. ff02fce